docs/src/content/docs/resources/glossary.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Glossary
+---
+
+import { LinkOut } from '@interledger/docs-design-system'
+
+## Account servicing entity (ASE)
+
+An account servicing entity provides and maintains a payment account for a payer and payee, and is a regulated entity in the country/countries it operates.
+
+## Authorization server (AS)
+
+An authorization server issues tokens to clients, which can then be used to perform authorized actions on resource servers. In the context of Open Payments, the authorization server grants permission for a client to access the Open Payments APIs and the `incoming-payment`, `quote`, and `outgoing-payment` resources. Open Payments leverages GNAP as the mechanism for delegating authorization.
+
+## Client
+
+A client is an application or service that interacts with the authorization server to obtain grants and tokens. A client uses these tokens to access resources or perform transactions on behalf of a user or system.
```

Comment:
I don't know if this is overkill. Just a suggestion. 

A client is an application or service, such as a mobile or web app, that interacts with an authorization server to obtain grants and tokens. A client uses these tokens to access resources on a resource server to perform actions, such as retrieving transaction history and setting up payments, on behalf of a user or system.